app-editors/featherpad/featherpad-0.18.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg qmake-utils
```

Comment:
Since *qmake-utils.eclass* doesn't import any functions, you can just sort this list alphabetically and it should work.